spec/stack_master/template_compilers/sparkle_formation_spec.rb

```diff
@@ -18,5 +19,19 @@ def compile
       compile
       expect(SparkleFormation.sparkle_path).to eq File.dirname(template_file_path)
     end
+
+    context 'with a custom sparkle_path' do
+      let(:compiler_options)  { { "sparkle_path" => '../foo' } }
+
+      it 'does not use the default path' do
+        compile
+        expect(SparkleFormation.sparkle_path).to_not eq File.dirname(template_file_path)
+      end
+
+      it 'expands the given path' do
+        compile
+        expect(SparkleFormation.sparkle_path).to match /^\/.+\/foo/
```

Comment:
The regex would be slightly more readable using `%r{}` and avoid needing to escape those forward slashes manually.